tests/schema_roundtrips.rs

```diff
@@ -29,25 +31,91 @@ fn roundtrip2(filename: &str) {
     assert_eq!(ast.to_string(), buf);
 }
 
-#[test] fn minimal() { roundtrip("minimal"); }
-#[test] fn scalar_type() { roundtrip("scalar_type"); }
-#[test] fn extend_scalar() { roundtrip("extend_scalar"); }
-#[test] fn minimal_type() { roundtrip("minimal_type"); }
-#[test] fn implements() { roundtrip("implements"); }
-#[test] fn implements_amp() { roundtrip2("implements_amp"); }
-#[test] fn implements_interface() { roundtrip("implements_interface"); }
-#[test] fn simple_object() { roundtrip("simple_object"); }
-#[test] fn extend_object() { roundtrip("extend_object"); }
-#[test] fn interface() { roundtrip("interface"); }
-#[test] fn extend_interface() { roundtrip("extend_interface"); }
-#[test] fn union() { roundtrip("union"); }
-#[test] fn empty_union() { roundtrip("empty_union"); }
-#[test] fn union_extension() { roundtrip("union_extension"); }
-#[test] fn enum_type() { roundtrip("enum"); }
-#[test] fn extend_enum() { roundtrip("extend_enum"); }
-#[test] fn input_type() { roundtrip("input_type"); }
-#[test] fn extend_input() { roundtrip2("extend_input"); }
-#[test] fn directive() { roundtrip("directive"); }
-#[test] fn kitchen_sink() { roundtrip2("kitchen-sink"); }
-#[test] fn directive_descriptions() { roundtrip2("directive_descriptions"); }
-#[test] fn repeatable() {roundtrip("repeatable")}
+#[test]
+fn minimal() {
+    roundtrip("minimal");
+}
+#[test]
+fn scalar_type() {
+    roundtrip("scalar_type");
+}
```

Comment:
This makes it only harder to read and add new. Alternative is probably to use `test_case` crate, but adding a dependency because we want to satisfy formatter? :(

Reply:
I strongly disagree with this sentiment. The new structure introduced by rustfmt is MUCH easier to read.

But ultimately the style does not matter much to me. What matters is consistency in formatting. And since all rust crates are formatted that way it's the good thing to do :)